buzzmobile/package.xml

```diff
@@ -16,6 +16,7 @@
   <build_depend>sensor_msgs</build_depend>
   <build_depend>cv_bridge</build_depend>
   <build_depend>message_generation</build_depend>
+  <build_depend>include</build_depend>
```

Comment:
If we will be deleting `buzzmobile/include/` do we need the changes in this file?